src/controllers/moderationactions/ModerationAction.cpp

```diff
@@ -124,6 +100,12 @@ ModerationAction::ModerationAction(const QString &action)
         this->line1_ = xD.mid(0, 2);
         this->line2_ = xD.mid(2, 2);
     }
+
+    if (!iconPath.isEmpty())
```

Comment:
Should probably use [QUrl::isValid](https://doc.qt.io/qt-6/qurl.html#isValid) here instead